nixos/modules/security/acme.nix

```diff
@@ -21,15 +21,41 @@ let
   # The Group can vary depending on what the user has specified in
   # security.acme.certs.<cert>.group on some of the services.
   commonServiceConfig = {
-      Type = "oneshot";
-      User = "acme";
-      Group = mkDefault "acme";
-      UMask = 0022;
-      StateDirectoryMode = 750;
-      ProtectSystem = "full";
-      PrivateTmp = true;
-
-      WorkingDirectory = "/tmp";
+    Type = "oneshot";
+    User = "acme";
+    Group = mkDefault "acme";
+    UMask = 0022;
```

Comment:
```
✗ UMask=                                                      Files created by service are world-readable by default                                                   0.1
```

I'm not sure, but can't be default to `0027` (640, 750) instead?

Reply:
Yeah I've done that in https://github.com/NixOS/nixpkgs/pull/121750/files.

Reply:
Sadly, this interferes with access to http-01 challenges

Reply:
Ah also #106603 which I almost forgot about!